R/extract.R

```diff
@@ -18,8 +18,8 @@ measr_extract <- function(model, ...) {
 #'     For `what = "odds_ratio_flags"` and
 #'     `what = "conditional_prob_flags"`, the compatibility interval used for
 #'     determining model fit flags to return. For example, a `ppmc_interval` of
-#'     0.95 (the default) will return any PPMCs where the posterior predictive
-#'     *p*-value (ppp) is less than 0.025 or greater than 0.975.
+#'     0.95 (the default) will return any \acronym{PPMC}s where the posterior
```

Comment:
All the acronyms have been wrapped in `\acronym{..}`. This tag is ignored by the linter.